### PR TITLE
Project update (taco.json update) should happen only if user chooses "y" when presented with the choice

### PR DESCRIPTION
**Pull request by meenakb**
1. Taco JSON should be updated only if user chose to update the project when presented with the choice
2. Fixing redundant status printing done when plugins are added/removed/updated

### CODE REVIEW

File: src/taco-cli/resources/en/resources.json [hunk 1]
```diff
@@ -170,6 +170,8 @@
     "_CommandKitSelectStatusUpdatingPlatforms.comment": "Status message on updating platforms",
     "CommandKitSelectStatusUpdatingPlugins": "Updating plugins...",
     "_CommandKitSelectStatusUpdatingPlugins.comment": "Status message on updating plugins",
+    "CommandKitSelectStatusUpdatingTacoJson": "Updating taco.json...",
```
Comment (nisheetjain) [inline src/taco-cli/resources/en/resources.json]: should this be full path to taco.json?

**Comment by nisheetjain** [comment]
do we have a test case covering this?

File: src/taco-cli/resources/en/resources.json (hunk 1 above)
Comment (meenakb) [inline src/taco-cli/resources/en/resources.json]: Soak and I prefer just "updating taco.json". More in line with "updating config.xml". what do you think?

**Comment by meenakb** [comment]
Test updates are pushed.

File: src/taco-cli/test/kit.ts [hunk 2]
```diff
@@ -299,7 +299,7 @@ describe("Kit command : ", function (): void {
 
         it("'taco kit select --cordova {CLI-VERSION}' on a project with a platform added, should execute with no errors", function (done: MochaDone): void {
             KitMod.yesOrNoHandler = getMockYesOrNoHandler(done, utils.emptyMethod, "PromptResponseNo");
-            runKitCommandSuccessCaseAndVerifyTacoJson(["select", "--cordova", "4.3.1"], tacoJsonPath, expectedCliTacoJsonKeyValues1)
+            runKitCommandSuccessCaseAndVerifyTacoJson(["select", "--cordova", "4.3.1"], tacoJsonPath, expectedKitTacoJsonKeyValues/*expectedCliTacoJsonKeyValues1*/)
```
Comment (nisheetjain) [inline src/taco-cli/test/kit.ts]: /*expectedKitTacoJsonKeyValues*/ not needed

File: src/taco-cli/test/kit.ts [hunk 3]
```diff
@@ -340,19 +340,19 @@ describe("Kit command : ", function (): void {
...
+                return runKitCommandSuccessCaseAndVerifyTacoJson(["select", "--kit", "5.1.1-Kit"], tacoJsonPath, expectedKitTacoJsonKeyValues);
+            })
+            .then(function(telemetryParameters: TacoUtility.ICommandTelemetryProperties): void {
+                var expected = {
+                    subCommand: { isPii: false, value: "select" },
+                    "options.kit": { isPii: false, value: "5.1.1-Kit" }
+                };
+                telemetryParameters.should.be.eql(expected);
+            }).then(() => {
+                TestProjectHelper.checkPlatformVersions(expectedKitPlatformVersion, cliProjectpath);
```
Comment (nisheetjain) [inline src/taco-cli/test/kit.ts]: can we add verify for plugin versions as well..

**Comment by nisheetjain** [comment]
:+1:

**Pull request merged**